.github/workflows/main.yml

```
@@ -0,0 +1,41 @@
+on:
+  pull_request: ~
+  push:
+    branches:
+      - master
+
+jobs:
+  tests:
+    name: Test (PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }})
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - php: 7.4
+            symfony: "^4.4"
+            coverage: true
+          - php: 8.1
+            symfony: "^4.4"
+          - php: 8.3
+            symfony: "^4.4"
```

No 8.2? It's just to save resources?
```suggestion
          - php: 8.1
            symfony: "^4.4"
          - php: 8.2
            symfony: "^4.4"
          - php: 8.3
            symfony: "^4.4"
```